client/public/apiary-proxy.js

```diff
@@ -2,9 +2,11 @@ const instance = axios.create({
   validateStatus: () => true
 });
 
-const Apiary = {};
+const Apiary = window.Apiary || {};
+Apiary.channels = [];
 
-Apiary.buildChannel = function (origin, scope) {
+//This function is exposed for testing purposes.
+const buildChannel = Apiary.buildChannel = function buildChannel(origin, scope) {
```

Comment:
Don't hate me, I don't find many reasons to use it in my day-to-day life. Gotta take my chance one when I see one.